extras/jenkins/DigitalOcean/Jenkinsfile

```diff
@@ -194,11 +188,11 @@ pipeline {
            */
 
           sh '''
-              # Destroy our partial build...
-              $WORKSPACE/bin/destroy.sh || true
-              # Clean up the Pulumi stack if it exists for our run - which it shouldn\'t, but you never know.
-              find  $WORKSPACE -mindepth 2 -maxdepth 7 -type f -name Pulumi.yaml -execdir $WORKSPACE/pulumi/python/venv/bin/pulumi stack rm marajenk${BUILD_NUMBER}  --force --yes  \;
+             # Destroy our partial build...
+             $WORKSPACE/bin/destroy.sh || true
+             # Clean up the Pulumi stack if it exists for our run - which it shouldn\'t, but you never know.
+             find  $WORKSPACE -mindepth 2 -maxdepth 7 -type f -name Pulumi.yaml -execdir $WORKSPACE/pulumi/python/venv/bin/pulumi stack rm marajenk${BUILD_NUMBER}  --force --yes  \\;
```

Comment:
Didn't we remove the double backslash in a previous PR?  Just want to make sure that this is what we want  Also curious what the double backslash means?

Reply:
This was an interesting journey. If you pick up that line with \\ and run it directly (like in the UDF scripts), the command fails because find wants to see `\;` at the end of the line. `\\;` escapes the `\` and leaves you with `;` which the shell interprets as the end of the line, and the `execdir` never runs.

So, what I noticed happening is stacks not being cleaned up, saw the `\\`, tested it locally, and thought "Aha! That is the issue! I shall fix that!"

Well, the fix worked for the scripting but bombed out in Jenkins. Becuase....the `\` is a reserved character in the Groovy scripting language, which is used by Jenkins.  Which then led me to the actual problem, which is that I was using `pulumi` without giving a path, and in Jenkins that meant "command not found" but we have a `|| true` in case it can't find anything.....

So, this should fix all the issues above, and if my past PR's are any indication I'm sure this will introduce another bug...